synapse/storage/databases/main/account_data.py

```diff
@@ -55,6 +56,64 @@
 logger = logging.getLogger(__name__)
 
 
+class InviteRule(Enum):
+    """Enum to define the sorting method used when returning destinations."""
+
+    ALLOW = "allow"
+    BLOCK = "block"
+
+
+class InviteRulesConfig:
+    default: InviteRule
+    user_exceptions: Dict[str, InviteRule]
+    server_exceptions: Dict[str, InviteRule]
+
+    def __init__(
+        self,
+        account_data: Optional[JsonMapping],
+        always_allow_user_id: Optional[str],
+    ):
+        account_data_safe = account_data or {}
+
+        default_str = account_data_safe.get("default")
```

Comment:
```suggestion
        default_str = account_data_safe.get("default") or "allow"
```

in case they've set `default: null`.